pkg/plugin/aggregation/update_test.go

```diff
@@ -47,3 +54,75 @@ func TestCreateUpdater(t *testing.T) {
 		t.Errorf("expected status to be failed, got %v", updater.status.Status)
 	}
 }
+
+func TestGetStatusPodName(t *testing.T) {
```

Comment:
👍 having the test. I just talked to someone who was suggesting we tweak the logic this method would use to take into account pod status (filter to only consider running pods). Something to consider, but regardless of the decision there, having the test will help us ensure that functionality.

Reply:
I'm happy to make that change but would prefer to do it in a follow up branch just to avoid adding another change in functionality here.

Reply:
Oh for sure; I didnt mean to do that here. Was just adding context for why I like having this test now (that it will immediately be useful for testing/adding requested functionality)